ard/api/interface.py

```diff
@@ -113,7 +113,7 @@ def set_up_ard_model(input_dict: Union[str, dict], root_data_path: str = None):
 def set_up_system_recursive(
     input_dict: dict,
     system_name: str = "top_level",
-    work_dir: str = "ard_prob_out",
+    work_dir: str = "case_files",
```

Comment:
do we want to make this dependent on the input files or case name some how? non-blocking